CWE/common/event_modifiers.txt

```diff
@@ -7888,3 +7904,22 @@ constitutional_crisis = {
 change_ruling_party = {
 	icon = 6
 }
+
+### Italian Special Statute Regions ###
```

Comment:
Please use the generic devolution modifier instead. Those modifiers have a system to simulate the stuff that these modifiers are aiming to do.